{"name": "test_3_verify_input_field_disabled_by_default", "status": "passed", "start": 1709527118741, "stop": 1709527118769, "uuid": "96a47ee4-f405-44aa-ba02-4604360bd43c", "historyId": "894a1829b02137c8e89258096dc964f0", "testCaseId": "894a1829b02137c8e89258096dc964f0", "fullName": "Tests.test_exception.TestLogin#test_3_verify_input_field_disabled_by_default", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_exception"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-UEVTOM8"}, {"name": "thread", "value": "21176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_exception"}]}